File: basic_algorithm/bubble_sort.py
def bubble_sort(arr):
  for j in range(len(arr) - 1):
    for i in range(len(arr) - 1):
      if arr[i] > arr[i + 1]:
        temp = arr[i]
        arr[i] = arr[i + 1]
        arr[i + 1] = temp
  return arr

print(bubble_sort([1,5,4,6,2,3,9,7]))

File: basic_algorithm/bubble_sort.js
function bubble_sort(arr) {
  for (let i = 0; i < arr.length - 1; ++i) {
    for (let j = i + 1; j < arr.length; ++j) {
      if (arr[i] > arr[j]) {
        let buff = arr[i];
        arr[i] = arr[j];
        arr[j] = buff;
      }
    }
  }

  return arr;
}

console.log(bubble_sort([4,39,9,5,65,34,23]));


